scripts/release.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e # exit with nonzero exit code if anything fails
+
+# exit with an error if the build fails
+if [[ ${TRAVIS_TEST_RESULT=0} == 1 ]]; then
+  exit 1;
+fi
+
+if [[ $TRAVIS_BRANCH == "master" ]]; then
+  # graduate the relase with --conventional-graduate
+  lerna version --conventional-commits --conventional-graduate --create-release github --yes
+  # publish the packages that were just versioned
+  lerna publish from-package --dist-tag next --yes
+  # push changes back to next so we can start work on the next release
+  git push origin next
```

Comment:
Will this have sufficient permissions?

Reply:
It'll be using the same git credentials travis has when it clones the repo to build it, so it _should_. Looking at Carbon Charts, they explicitly add a token - https://github.com/carbon-design-system/carbon-charts/blob/master/scripts/deploy.sh#L9-L11 - But I'm not 100% sure if that's necessary.

Reply:
I think I remember having to specifically add those lines bc travis wouldn't publish

Reply:
Cool, I'll add something similar

Reply:
We have both `next` and `master` configured as protected branches, so I don't think this will work. We just turned off travis in favor of a github action. It can be swapped out to run this script instead, but [actions are limited in the same way](https://github.community/t/allowing-github-actions-bot-to-push-to-protected-branch/16536/2).

We can do this manually for now and/or find a way to do it via the rest API as mentioned in that thread.

Reply:
Interesting 🤔 may be best if we do it manually for now and investigate our options after the switch